render-report.ts

```diff
@@ -129,7 +129,7 @@ export function renderReport(
         </div>
       </div>
       <p style="font-weight: 500; text-decoration: underline;">
-        ${numberRepresentation} - punt ${addLeadingZeros(agendaItemNumber, 4)}
+      ${reportParts.concerns}
```

Comment:
I think this is in the wrong repository? I assume this is about

> heel het stukje “Betreft” uit de piece-part van de report (indien die piece-part al bestaat) dient als titel gebruikt te worden, inclusief de documentnummers.

That should be in https://github.com/kanselarij-vlaanderen/minutes-report-generation-service

In any case KAS-4303 is only about the minutes generation, not the report generation.